### PR TITLE
Info2-lecture: Twhining quicksort implementation

### PR DESCRIPTION
**Pull request by Twhining**

### CODE REVIEW

**Comment by C0DECYCLE** [comment]
Very nice thank you! Please use "Info-2-...:" (... = lecture or 000-00-...) as a prefix for the commit messages and titles, otherwise LGTM.

**Comment by C0DECYCLE** [comment]
PS: The documentation label is for readme or wiki related stuff.

File: Informatics II/Lectures/Twihning/quicksort.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    int arr[10] = {6,3,30,22,12,5,14,9,7,4};
+    int start = 0;
+    int end = n;
+    quick_sort(arr, start, end);
+
+    for(int i=0;i<end;i++){
+        printf("%d ", arr[i]);
+    }
+    return 0;
+    }
```
Comment (C0DECYCLE) [inline Informatics II/Lectures/Twihning/quicksort.c:58]: please unindent

File: Informatics II/Lectures/Twihning/quicksort.c [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+void quick_sort(int *a,int s, int e){
+    int index;
+    if (s >= e){
+        return;
+    }
+    index = hoare_partition(a, s, e);
+    quick_sort(a, s, index - 1);
+    quick_sort(a,index + 1, e);
+
+    }
```
Comment (C0DECYCLE) [inline Informatics II/Lectures/Twihning/quicksort.c:45]: please unindent

**Review by C0DECYCLE — CHANGES_REQUESTED** [review]
Overall LGTM, just to minor formattings. @Patertuck would be awesome if you could double check!

File: Informatics II/Lectures/Twihning/quicksort.c (hunk 1 above)
Comment (Patertuck) [reply Informatics II/Lectures/Twihning/quicksort.c:58]: PS: You can unindent all selected lines by holding shift and pressing tab :)

**Review by Patertuck — APPROVED** [review]

**Review by C0DECYCLE — APPROVED** [review]
LGTM

**Pull request merged**